jena-arq/pom.xml

```diff
@@ -85,8 +84,9 @@
     </dependency>
 
     <dependency>
-      <groupId>com.github.jsonld-java</groupId>
-      <artifactId>jsonld-java</artifactId>
+       <groupId>com.github.jsonld-java</groupId>
+        <artifactId>jsonld-java</artifactId>
+        <version>0.8.3</version>
     </dependency>
```

Comment:
`<version>` not needed - the dependency management in jena-parent controls the version.